recipes/folly/all/conanfile.py

```diff
@@ -132,11 +132,10 @@ def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=False)
 
     def _cppstd_flag_value(self, cppstd):
-        if is_msvc(self):
-            prefix = "c"
-            year = str(cppstd)
-            if year > "17":
-                year = "latest"
+        prefix = "c"
```

Comment:
I wouldn't refactor anything because of a linter warning that does not apply (it doesn't apply because `_cppstd_flag_value` is only used already guarded by a conditional `is_msvc`) - so it's not "wrong", regardless of what the linter says.

Having `c++latest

If we really want to refactor this: turn this into a variable used here instead - that way its localised (otherwise it looks strange to have a `_cppstd_flag_value` that would have `c++latest` as a value, which is only accepted by msvc.

https://github.com/conan-io/conan-center-index/blob/19e213670100fa4a3cfd0abdf83b31d600039fe7/recipes/folly/all/conanfile.py#L171-L172